contrib_bots/bot_lib.py

```diff
@@ -77,10 +78,49 @@ def get_state(self):
     if not quiet:
         print(message_handler.usage())
 
+    def is_bot_called(message, client):
+        bot_mention = r'^@(\*\*{0}\*\*\s|{0}\s)(?=.*)'.format(client.full_name)
+        start_with_mention = re.compile(bot_mention).match(message['content'])
+        if start_with_mention:
+            query = message['content'][len(start_with_mention.group()):]
+            return query
+        else:
+            bot_response = 'Please mention me first, then type the query.'
```

Comment:
I think we want to rename this function, since it doesn't return a boolean value, but instead it returns a message or has side effects.  I would maybe call it something like `extract_message_if_mentioned`.  Or maybe try to break it up somehow.